### ISSUE TITLE
If `body` is a string, the result of the cache is problematic.

### ISSUE THREAD

**Issue by zfowed** [mkozjak/koa-cache-lite#34]
ps: Forgive me for not going to English, all Google Translate.

I tried to assign an HTML string to `ctx.body`, but the cached result does not meet my expectations.

my code:

```javascript
const convert = require('koa-convert');
const cache = require('koa-cache-lite');

cache.configure({
  '/': 3000
}, {
  debug: true
})

app.use(convert(cache.middleware()));

app.use(function (ctx, next) {
  ctx.body = "<!DOCTYPE html><html><head><title>DEMO</title></head><body><div class="my-app"></div></body></html>"
});

```
cache results:

```html
"<!DOCTYPE html><html><head><title>DEMO</title></head><body><div class=\"my-app\"></div></body></html>"
```

I found this when reading the source code

```javascript
storeRequest[requestKeyBody] = JSON.stringify(_response_body)
```

**Comment by mkozjak** [comment]
Use `ctx.body = '<!DOCTYPE html><html><head><title>DEMO</title></head><body><div class="my-app"></div></body></html>'` instead of `ctx.body = "<!DOCTYPE html><html><head><title>DEMO</title></head><body><div class="my-app"></div></body></html>"` (single ticks instead of double ticks)

**Comment by zfowed** [comment]
Forgive me for writing my code wrong, but even then there is still this problem

**Comment by zfowed** [comment]
Forgive me for writing my code wrong, but even then there is still this problem
![image](https://user-images.githubusercontent.com/23629388/39508908-c23267dc-4e16-11e8-8b25-d29e362d5048.png)

**Comment by mkozjak** [comment]
True! Thanks for reporting! Fixed in 3.8.5.
